TYPO3.Neos/Resources/Public/JavaScript/Content/Components/TargetWorkspaceSelector.js

```diff
@@ -22,20 +22,20 @@ define(
       classNameBindings: ['open:neos-open'],
       template: Ember.Handlebars.compile(template),
 
-      controller: TargetWorkspaceController,
-      nodeSelection: NodeSelection,
-      publishableNodes: PublishableNodes,
+			_controller: TargetWorkspaceController,
```

Comment:
had weird issues when using the ``controller`` variable, some magic got in the way.. can't remember exactly what though